test/specs/modules/Rating/Rating-test.js

```diff
@@ -1,190 +1,199 @@
 import _ from 'lodash'
 import React from 'react'
 
-import Rating from 'src/modules/Rating/Rating'
 import * as common from 'test/specs/commonTests'
 import { sandbox } from 'test/utils'
+import Rating from 'src/modules/Rating/Rating'
 
 describe('Rating', () => {
   common.isConformant(Rating)
   common.hasUIClassName(Rating)
 
+  common.propKeyOnlyToClassName(Rating, 'disabled')
+
   common.propValueOnlyToClassName(Rating, 'size')
   common.propValueOnlyToClassName(Rating, 'icon')
-  common.propKeyOnlyToClassName(Rating, 'disabled')
 
   describe('clicking on icons', () => {
     it('makes icons active up to and including the clicked icon', () => {
       const wrapper = mount(<Rating maxRating={3} />)
-      const icons = wrapper.find('.icon')
+      const icons = wrapper.find('RatingIcon')
 
       icons.at(1).simulate('click')
 
-      icons.at(0).should.have.className('active')
-      icons.at(1).should.have.className('active')
-      icons.at(2).should.not.have.className('active')
+      icons.at(0).should.have.prop('active', true)
+      icons.at(1).should.have.prop('active', true)
+      icons.at(2).should.have.prop('active', false)
     })
 
-    it('removes the "selected" class', () => {
+    it('removes the "selected" prop', () => {
       const wrapper = mount(<Rating maxRating={3} />)
-      const icons = wrapper.find('.icon')
+      const icons = wrapper.find('RatingIcon')
 
-      icons
-        .last()
+      icons.last()
         .simulate('mouseEnter')
         .simulate('click')
 
-      icons.every('.icon.selected')
-        .should.equal(false, 'Some icon did not remove its "selected" class')
-
       wrapper.should.not.have.className('selected')
+
+      icons.findWhere((i) => i.prop('selected', true))
+        .should.have.length(0)
```

Comment:
Would prefer to keep the custom failure messages in these assertions.  Otherwise, a failed test will just say `Expected array to have length of 0, but it was X`.  Whereas, the custom message is more helpful `Some icon did not remove its "selected" prop`